src/components/Autocomplete/AutocompleteHit.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useRef, useEffect } from 'react';
+import classname from 'classnames';
+import classNames from '../../core/config/ClassNames';
+import { AutocompleteHitInterface } from '@paper/components/Autocomplete/interfaces/AutocompleteHitInterface';
+
+export const AutocompleteHit: React.FunctionComponent<AutocompleteHitInterface> = ({
+  active,
+  children = null,
+  onClick,
+  scrollTo,
+}) => {
+  const ref: React.MutableRefObject<any> = useRef();
```

Comment:
```suggestion
  const ref = useRef<HTMLDivElement>();
```

If you specify it like this, it will use the type returned by useRef.